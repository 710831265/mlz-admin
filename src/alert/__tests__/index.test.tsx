import React, { Component } from 'react';
import Alert from '..';
import testMount from '../../../tests/testMount';
import testSnapshot from '../../../tests/testSnapshot';

describe('ðŸ§ª Alert', () => {
  testMount(Alert);
  testSnapshot(Alert);
});
