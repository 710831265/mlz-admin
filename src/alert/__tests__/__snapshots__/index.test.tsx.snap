// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`🧪 Alert snapshot test 快照匹配 1`] = `"<div data-show=\\"true\\" class=\\"ant-alert ant-alert-info ant-alert-no-icon\\" role=\\"alert\\"><div class=\\"ant-alert-content\\"><div class=\\"ant-alert-message\\"></div><div class=\\"ant-alert-description\\"></div></div></div>"`;
